showcase/tests/integration/components/hds/pagination/compact-test.js

```diff
@@ -153,12 +153,30 @@ module('Integration | Component | hds/pagination/compact', function (hooks) {
 
   test('it should throw an assertion if @queryFunction is not a function', async function (assert) {
     const errorMessage =
-      '@queryFunction for "Hds::Pagination::Numbered" must be a function';
+      '@queryFunction for "Hds::Pagination::Compact" must be a function';
     assert.expect(2);
     setupOnerror(function (error) {
       assert.strictEqual(error.message, `Assertion Failed: ${errorMessage}`);
     });
-    await render(hbs`<Hds::Pagination::Compact @queryFunction="foo" />`);
+    await render(
+      hbs`<Hds::Pagination::Compact @queryFunction="foo" @model="test" />`
+    );
+    assert.throws(function () {
+      throw new Error(errorMessage);
+    });
+  });
+
+  test('it should throw an assertion if @queryFunction is provided without a routing argument', async function (assert) {
```

Comment:
Is there a test that it doesnt throw an assertion if it is uncontrolled and doesn't have a routing argument?

thinking about it more, I guess if the other tests work then thats implied...